ui/src/utils/index.tsx

```diff
@@ -140,6 +140,14 @@ export const copy = (text: string | null | undefined) => {
   showSuccessAlert('Copied')
 }
 
+/**
+ * Method to copy to clipboard a given strings array
+ * @param array Strings array to process
+ */
+export const copyArrayToClipboard = (array: string[] = []) => {
+  copy(array.join('\n'))
```

Comment:
Is this somewhere it could make sense to use the PapaParse library?

Reply:
No so sure, I just was trying use `unparse` but seems it does not work for array of strings.